openhie/package-starter-kit
# CLI-V2 pre-prod #1

Don't review yet, these PRs are first being structured in the most convenient fashion.

### What this PR does
- add root level cli commands

### Testing
- ensure you have Go 1.18 onwards
- run CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o clix; ./clix --help; rm clix
- review the commands that have been added
